contentgrid-spring-boot-platform/build.gradle

```diff
@@ -13,7 +13,9 @@ ext {
 
 dependencies {
     api platform("org.springframework.boot:spring-boot-dependencies:2.6.12")
-    api platform("com.contentgrid.thunx:thunx-dependencies:0.4.0")
+    api platform("com.contentgrid.thunx:thunx-bom:0.4.2")
+
+    api "org.springframework.boot:spring-boot-actuator"
```

Comment:
This is a platform, direct dependencies should only be other platforms, not actual dependencies.

The rest of version constraints should be under `constraints`.

In this case, the `spring-boot-actuator` version is already set by the `spring-boot-dependencies` platform, so it is not necessary at all.